src/app/components/gallery-elements-bar/gallery-elements-bar.component.ts

```diff
@@ -1,4 +1,10 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { queryParams } from 'src/app/utils/consts/query-params';
+import * as activatedRouteUtil from 'src/app/utils/other/activated-route-util';
```

Comment:
Maybe it's better to import data like 'import { getQueryParamValue } from ... '?